srv/salt/ceph/rgw/files/users.j2

```diff
@@ -0,0 +1,11 @@
+
+realm:
+  default:
+{% if salt['file.file_exists']('/srv/salt/ceph/rgw/users/system.user.yml') %}
+{% include "ceph/rgw/users/system.user.yml" %}
```

Comment:
Why do you want to support more than one location for the system user info?
It's better if `system.user.yml` is only allowed to be in a single location

Reply:
Because I do not want the admin modifying files supplied by the rpm.  Upgrades are bad enough in general.  

If the admin does not care for our name, etc., then all of their changes are under the rgw/users directory.

Reply:
Makes sense, we just need to make sure that this is properly documented then.